pkg/option/runtime_options.go

```diff
@@ -60,6 +61,11 @@ var (
 		Description: "Enable trace notifications",
 	}
 
+	specPolicyTracingAccounting = Option{
```

Comment:
I guess that it should be "specPolicyAccounting" rather than "specPolicyTracingAccounting".

Reply:
By the way, you put "rebase"  as the commit message. It seems that there is something wrong in your workspace causing it.

Reply:
@LeiW000 thanks review i'll fixed it, I'm struggling with the part of the current option that writes to the header file because I don't quite understand it. Can you help me?

Reply:
Yes, it seems that you forgot to map specPolicyAccounting to PolicyAccounting. I just created a patch. You can cherry-pick it to try. 
https://github.com/LeiW000/cilium/commits/policy_accounting
193796b0613b4db0e8bc1da6f5b5c3419d37f84d

Reply:
@Jack-R-lantern , could you give me write permission for your branch[ A-Performance-issue-with-calling-__sync_fetch_and_add](https://github.com/Jack-R-lantern/cilium/tree/ISSUE-28487/A-Performance-issue-with-calling-__sync_fetch_and_add)? Then, I can push my commit into this PR directly. 

By the way, the options are written into /run/cilium/state/[endpoint id]/ ep_config.h, which will be used to compile the bpf program.

Reply:
@LeiW000 Sure I invite you to